docs/src/main/sphinx/release/release-398.md

```diff
@@ -0,0 +1,61 @@
+# Release 398 (28 Sep 2022)
+
+## General
+
+* Add Hudi connector. ({issue}`10228`)
+* Add metrics for the execution time of filters and projections to `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`14135`)
+* Show local cost estimates when using `EXPLAIN`. ({issue}`14268`)
+* Fix timeouts happening too early because of improper handling of the
+  `node-scheduler.allowed-no-matching-node-period` configuration property. ({issue}`14256`)
+* Fix query failure for `MERGE` queries when `task_writer_count` is greater
+  than one. ({issue}`14306`)
+
+## Accumulo connector
+
+* Add support for column comments when creating a new table. ({issue}`14114`)
+* Add column mapping and index information in an extra field instead of a
+  comment. ({issue}`14095`)
+
+## BigQuery connector
+
+* Fix improper escaping of backslash and newline characters. ({issue}`14254`)
+* Fix query failure when the predicate involves a varchar value with a
+  backslash. ({issue}`14254`)
+
+## ClickHouse connector
+
+* Upgrade minimum required version to 21.8. ({issue}`14112`)
+
+## Delta Lake connector
+
+* Improve performance when reading Parquet files for queries with predicates. ({issue}`14247`)
+
+## Elasticsearch connector
+
+* Deprecate the use of prefix-based pass-through queries in favor of the
+  `raw_query` table function. Legacy behavior can be re-enabled with the
+  `elasticsearch.legacy-pass-through-query.enabled` configuration property. ({issue}`14015`)
+
+## Hive connector
+
+* Add support for partitioned views when using legacy mode with Hive views. ({issue}`14028`)
```

Comment:
legacy mode of what?

Reply:
`hive.hive-views.legacy-translation` https://trino.io/docs/current/connector/hive.html#hive-views

Reply:
Ok, then maybe:

> Add support for partitioned views when legacy mode for view translation is enabled.